src/plugins/spectrogram.ts

```
@@ -211,6 +211,14 @@ function FFT(bufferSize: number, sampleRate: number, windowFunc: string, alpha:
   }
 }
 
+// Add typings for FFT
+declare class FFT {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'FFT'. Consider to delete it or rename it.

'FFT' is defined here:


(lint/suspicious/noRedeclare)